components/datepicker/metadata/mods.md

```diff
@@ -0,0 +1,38 @@
+| Modifiable Custom Properties                            |
+| ------------------------------------------------------- |
+| `--mod-datepicker-border-color-disabled`                |
+| `--mod-datepicker-border-radius`                        |
+| `--mod-datepicker-border-radius-quiet`                  |
+| `--mod-datepicker-dash-color-disabled`                  |
+| `--mod-datepicker-dash-font-size`                       |
+| `--mod-datepicker-dash-line-height`                     |
+| `--mod-datepicker-datetime-input-width`                 |
+| `--mod-datepicker-datetime-input-width-first`           |
+| `--mod-datepicker-datetime-quiet-input-width`           |
+| `--mod-datepicker-datetime-quiet-input-width-first`     |
+| `--mod-datepicker-focus-animation`                      |
+| `--mod-datepicker-focus-line-gap`                       |
+| `--mod-datepicker-focus-ring-color`                     |
+| `--mod-datepicker-focus-ring-gap`                       |
+| `--mod-datepicker-focus-ring-width`                     |
+| `--mod-datepicker-focus-thickness`                      |
+| `--mod-datepicker-generic-padding`                      |
+| `--mod-datepicker-input-width`                          |
+| `--mod-datepicker-input-width-quiet`                    |
+| `--mod-datepicker-invalid-quiet-color`                  |
+| `--mod-datepicker-min-width`                            |
+| `--mod-datepicker-padding-inline`                       |
+| `--mod-datepicker-padding-inline-end-invalid`           |
+| `--mod-datepicker-padding-inline-end-invalid-quiet`     |
+| `--mod-datepicker-padding-inline-end-quiet`             |
+| `--mod-datepicker-pickerbutton-border-color`            |
+| `--mod-datepicker-pickerbutton-border-color-invalid`    |
+| `--mod-datepicker-quiet-border-color-hover`             |
+| `--mod-datepicker-quiet-button-offset`                  |
+| `--mod-datepicker-range-dash-margin-left`               |
+| `--mod-datepicker-range-dash-padding-top`               |
+| `--mod-datepicker-range-input-width-first`              |
+| `--mod-datepicker-range-input-width-quiet-first`        |
+| `--mod-picker-button-border-color`                      |
+| `--mod-textfield-icon-spacing-inline-end-invalid`       |
```

Comment:
If you want to run this build again, it should update the mods file to remove the passthroughs